zilliqa/src/api/admin.rs

```diff
@@ -24,6 +24,7 @@ pub fn rpc_module(
             ("admin_consensusInfo", consensus_info),
             ("admin_generateCheckpoint", checkpoint),
             ("admin_blockRange", admin_block_range),
+            ("admin_forceSetView", force_set_view),
```

Comment:
nit: I don't like that this has 2 verbs. Is `admin_forceView` or `admin_setView` okay?